namespaces/live-1.cloud-platform.service.justice.gov.uk/money-to-prisoners-prod/02-limitrange.yaml

```diff
@@ -9,6 +10,6 @@ spec:
         cpu: 250m
         memory: 500Mi
       defaultRequest:
-        cpu: 10m
-        memory: 100Mi
+        cpu: 125m
```

Comment:
This is too high. Please set cpu defaultRequest to no more than 10m

NB: This is a *reservation* not any kind of limit.